test/connector/http/basicauth/test

```diff
@@ -16,6 +16,8 @@ if $local_test; then
   docker_volume_args=(--volume "$(cd ../../../..; pwd):/secretless")
 fi
 
+export isError=false
```

Comment:
There is no need for this to be exportable. Also, use string values (`"true"` and `"false"`) as it can make the comparison _very_ ambiguous depending on the conditional checks.